packages/vega-view-transforms/src/layout/legend.js

```diff
@@ -26,6 +26,7 @@ export function legendParams(g, orient, config, xb, yb, w, h) {
   const _ = lookup(config, orient),
         offset = offsets(g, _('offset', 0)),
         anchor = _('anchor', Start),
+        frame = _('frame', Group),
```

Comment:
Thanks for the new param! For this to show up in the JsonSchema, I believe we'll need to add this to around here in the Typescript types: 

https://github.com/vega/vega/blob/7862e6db14ae2fb02b21444921d7de60160a36a2/packages/vega-typings/types/spec/config.d.ts#L586